hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestAbfsClient.java

```diff
@@ -240,4 +242,25 @@ public void verifyUserAgentClusterType() throws Exception {
       .contains(DEFAULT_VALUE_UNKNOWN);
   }
 
+  public static AbfsClient createTestClientFromCurrentContext(
```

Comment:
Same doubt as the above comment.
TestAbfsClient should have test methods the AbfsClient and supporting methods.
This method looks more like a Utility.

Reply:
Again, dont which to create a seperate test utlity class for AbfsClient alone to return a test instance and hence have placed it in the unit test class for AbfsClient.
For future tests that will need to mock or create new instances, it will be easy to check respective unit test class for any method available.